README.md

```diff
@@ -36,6 +36,9 @@ ec2-user hard nproc 16384
 EOL
 ```
 
+Note: In the above configuration, `ec2-user` it's a
```

Comment:
This change isn't related to updating Firecracker version and should be added in a separate commit.

Reply:
New commit added with this specifc change.